pkgs/top-level/all-packages.nix

```diff
@@ -13339,10 +13339,6 @@ with pkgs;
   cudatext-gtk = callPackage ../applications/editors/cudatext { widgetset = "gtk2"; };
   cudatext = cudatext-qt;
 
-  comical = callPackage ../applications/graphics/comical {
```

Comment:
you removed comical from pkgs, why?

Reply:
`comical` is still in `pkgs`, it just doesn't need an explicit `callPackage` if the expression is in `pkgs/by-name`. Or have I missed something?

Reply:
thats true, I missed the renaming from pkgs/applications to pkgs/by-name